.github/workflows/ci.yml

```diff
@@ -39,6 +50,11 @@ jobs:
       - name: Submodules
         run: |
           git submodule update --init --recursive
+      - name: Run examples
+        run: |
+          PGPASSWORD=postgres psql -U postgres -h localhost postgres -c 'CREATE DATABASE pgml'
```

Comment:
The extension needs to be installed into Postgres system folders for this to work.

```
export POSTGRES_VERSION=14
cargo pgx package
# Copy the extension .so
sudo cp target/release/pgml-pg${POSTGRES_VERSION}/usr/lib/postgresql/${POSTGRES_VERSION}/lib/* \
   /usr/lib/postgresql/${POSTGRES_VERSION}/lib

# Copy the control & SQL files
sudo cp target/release/pgml-pg${POSTGRES_VERSION}/usr/share/postgresql/${POSTGRES_VERSION}/extension/* \
   /usr/share/postgresql/${POSTGRES_VERSION}/extension
```